apps/atomy-q/API/app/Adapters/Ai/ConfiguredAiEndpointRegistry.php

```
@@ -55,6 +55,8 @@ public function endpointConfig(string $endpointGroup): ?AiEndpointConfig
         $provider = $this->providerSection();
         $defaultAuthToken = trim((string) ($provider['default_auth_token'] ?? ''));
         $defaultTimeout = (int) ($provider['default_timeout_seconds'] ?? 10);
+        $defaultRetryAttempts = (int) ($provider['default_retry_attempts'] ?? 1);
+        $defaultRetryBackoffMs = (int) ($provider['default_retry_backoff_ms'] ?? 0);
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $defaultRetryBackoffMs. Keep variable name length under 20. (undefined)

(LongVariable)